example/app/data/DemoDataSource/apps/using-remote-package/blueprints/RemoteExample.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "RemoteExample",
+  "type": "CORE:Blueprint",
+  "attributes": [
+    {
```

Comment:
Just a comment, decide if you want to change this:

if you extend from CORE:NamedEntity, you can remove the "type", "name" and "description" attributes form this Blueprint.